README.md

```diff
@@ -56,34 +56,36 @@ interface SourceInformation {
 }
 ```
 
-* getBasket(reference, [channelId], returnTTId) ⇒ Basket
-* createBasket(basketData, returnTTId) ⇒ Basket
-* getDeliveries(basketReference, basketItems, [channelId]) ⇒ Array&lt;Delivery&gt;
-* setSelectedDelivery(basket, selectedDelivery) ⇒ Basket
-* addItems(basket, basketItems, returnTTId) ⇒ Basket
-* replaceItems(basket, basketItems, returnTTId) ⇒ Basket
+* getBasket(reference, checksum, [channelId], returnTTId) ⇒ Basket
+* createBasket(basketData, checksum, returnTTId) ⇒ Basket
```

Comment:
How would we have a checksum before creating a basket?

Reply:
yes good point, I'll remove that one. Thanks!